applications/StructuralMechanicsApplication/python_scripts/python_solvers_wrapper_structural.py

```diff
@@ -5,8 +5,8 @@
 
 def CreateSolver(model, custom_settings):
 
-    if (type(main_model_part) != KratosMultiphysics.ModelPartInterface):
-        raise Exception("input is expected to be provided as a Kratos ModelPart object")
+    if (type(model) != KratosMultiphysics.Model):
```

Comment:
are u sure that this works? I think it should be ModelInterface...

did u compile this locally before making the PR?

Reply:
ok i fixed it and pushed the fix. take care to ran the tests next time you merge

Reply:
This is due to the merge with the master

Reply:
I added ModelPartInterface